docs/reST/tut/tom_games3.rst

```diff
@@ -71,15 +71,15 @@ control over your resources. Take this example of an image loading function::
 
   def load_png(name):
       """ Load image and return image object"""
-      fullname = os.path.join('data', name)
+      fullname = os.path.join("data", name)
       try:
           image = pygame.image.load(fullname)
           if image.get_alpha() is None:
               image = image.convert()
           else:
               image = image.convert_alpha()
       except pygame.error, message:
```

Comment:
This code is wrong as well. Pygame.image.load emits FileNotFoundError, not pygame.error. And what is this message thing? Python3 gives me a syntax error from that.